app/models/product.rb

```diff
@@ -140,6 +142,18 @@ def add_filtering_tests
     ApplicationController.helpers.generate_filter_records(filter_tests)
   end
 
+  # When the 2014 certification edition is enabled, duplicate_records is always disabled.
+  # Since the default for duplicate_records is true, we override it here. We also make sure
+  # that c4_test is not somehow set to true.
+  def enforce_cert_edition_settings
+    if cert_edition.eql? '2014'
+      self.duplicate_records = false
+      self.c4_test = false
+    end
+
+    true
```

Comment:
What is the purpose of this `true` return statement here?

Reply:
before_* always have to return true, if that is not there then `self.c4_test = false` is that last statement which makes the return code `false`.